### PR TITLE
Add invoices list to the billing page

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Reshuffle subscription fields
Update spinner location and logic

[stage-2]

## Motivation and Context
Billing Frictions epic

## How Has This Been Tested?
Tested changes locally. Will continue to make PRs on this branch before releasing.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc fyi, this is working branch/PR for the invoices list. Will fix unit tests and various issues in subsequent PRs. Thanks.

File: web/scripts/billing/controllers/ctr-billing.js [hunk 1]
```diff
@@ -110,29 +90,26 @@ angular.module('risevision.apps.billing.controllers')
         return subscription.status === 'Suspended';
       };
 
-      function _reloadSubscriptions() {
-        $timeout(function () {
-          $loading.startGlobal('subscriptions-changed-loader');
```
Comment (alex-deaconu) [inline web/scripts/billing/controllers/ctr-billing.js:115]: @ezequielc I'm not sure why we were showing a global spinner for this refresh. I've updated to use the local spinner, let me know if you disagree. Thanks.

File: web/partials/billing/app-billing.html [hunk 2]
```diff
@@ -97,10 +88,53 @@ <h1 class="app-header-title">Account & Billing</h1>
...
+              <a class="madero-link u_clickable">
+                <img src="../images/icon-download.svg" width="20" height="20">
+              </a>
+            </td>
+            <td class="table-body__cell">
+              <a class="madero-link u_clickable" ng-if="invoice.status !== 'PAID'" ng-href="{{invoice.chargebeePayNowUrl}}" target="_blank">Pay Now</a>
+            </td>
+          </tr>
+
+          <tr ng-show="invoices.items.list.length === 0 && uninvoices.items.list.length === 0">
```
Comment (ezequielc) [inline web/partials/billing/app-billing.html:132]: @alex-deaconu The empty list message is not showing, looks like you are looking for `uninvoices`, which I don't think exists. Thanks!

**Review by ezequielc — APPROVED** [review]
@alex-deaconu Looks good, there is one minor issue inline, but can be fixed before merging to master. Thanks!

File: web/partials/billing/app-billing.html (hunk 2 above)
Comment (alex-deaconu) [reply web/partials/billing/app-billing.html:132]: @ezequielc copy paste error, good catch!

**Comment by alex-deaconu** [comment]
@ezequielc we're now hiding the Pay Now button via the latest commit. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Looks good. 👍  Thanks for updating it!

**Pull request merged**